README.md

```diff
@@ -30,4 +30,51 @@ source migration-env/bin/activate
 python3 -m pip install -r requirements.txt
 ```
 
-Then start the migration script `python3 migrate.py`.
\ No newline at end of file
+Then start the migration script `python3 migrate.py`.
```

Comment:
please undo the changes in the readme - the change to migrate.py is sufficient

Reply:
I have made the necessary modifications